tools/src/launcher/Main.hs

```diff
@@ -658,12 +642,11 @@ getSafeModeConfigPath :: LauncherOptions -> FilePath
 getSafeModeConfigPath LO{..} = loStatePath </> "safemode.yaml"
 
 runWallet
-    :: Bool              -- ^ wallet logging
-    -> NodeData          -- ^ Wallet, its args, wallet log file
+    :: NodeData          -- ^ Wallet, its args, wallet log file
     -> Maybe FilePath    -- ^ Node log file
     -> LauncherOptions   -- ^ Launcher Options
     -> M ExitCode
-runWallet shouldLog nd nLogPath lo = do
+runWallet nd nLogPath lo = do
     safeMode <- readSafeMode lo
     let wpath = ndPath nd
         wargs = (ndArgs nd) <> (if safeMode then [ "--safe-mode", "--disable-gpu", "--disable-d3d11" ] else [])
```

Comment:
more options turned off.

Reply:
nikola requested that safemode send more flags